functions.php

```diff
@@ -4,10 +4,27 @@
  * Additional code for the child theme goes in here.
  */
 
-add_action( 'wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_head', 'hook_javascript');
 
-function enqueue_child_styles() {
+function enqueue_child_styles()
+{
 	$css_creation = filectime(get_stylesheet_directory() . '/style.css');
 
-	wp_enqueue_style( 'child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation );
+	wp_enqueue_style('child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation);
+}
+
+function hook_javascript()
+{
+?>
+<!-- Fundraise Up: world-class checkout experience for serious online fundraising -->
```

Comment:
Unless we're getting paid to advertise, we can save our supporters a few bytes of bandwidth 😄 

```suggestion
<!-- Begin Fundraise Up -->
```